net/oic/pkg.yml

```diff
@@ -23,11 +23,15 @@ pkg.author: "https://github.com/iotivity/iotivity-constrained"
 pkg.homepage: "https://www.iotivity.org/"
 pkg.keywords:
 
+pkg.apis:
+    - TRNG_HW_IMPL
```

Comment:
This doesn't seem quite right.  The `net/oic` package does not implement the random number generator interface does it?

Reply:
No, net/oic will need to use `trng`. Not sure if I am setting the dependency right. If I don't, I hit
`Error: Unsatisfied APIs detected:
    * TRNG_HW_IMPL, required by: hw/drivers/trng`